scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -100,6 +100,23 @@ def get_parameter_default(param: inspect.Parameter) -> None:
 
 
 def get_property_names(node: ast.AST, attribute_names: Dict) -> None:
+    assign_nodes = [node for node in node.body if isinstance(node, ast.AnnAssign)]
+    # Check for class level attributes that follow the pattern: foo: List["_models.FooItem"] = rest_field(name="foo")
+    for assign in assign_nodes:
+        if hasattr(assign, "target"):
+            if hasattr(assign.target, "id") and not assign.target.id.startswith("_"):
+                attr = assign.target.id
+                attr_type = None
+                # FIXME: This can get the type hint for a limited set attributes. We need to address more complex
```

Comment:
I wonder if there's some logic from APIview that we could borrow for this

Reply:
That would be great, I'll make sure to take a look!